### PR TITLE
DPT-948 Support new query schema type

### PR DESCRIPTION
**Pull request by arcneon**

### CODE REVIEW

File: basestar-graphql/src/test/resources/io/basestar/graphql/schema.yml [hunk 1]
```diff
@@ -123,3 +123,20 @@ Function:
   returns: boolean
   definition: ""
 
+NearPoints:
+  type: query
+  sql: SELECT X, Y FROM ${point}
```
Comment (andriusvelykis) [inline basestar-graphql/src/test/resources/io/basestar/graphql/schema.yml:128]: The query does not seem to be making use of the arguments

File: basestar-schema/src/main/java/io/basestar/schema/from/FromSql.java [hunk 2]
```diff
@@ -168,4 +169,26 @@ public String getReplacedSql(final Function<Schema<?>, String> replacer) {
...
+
+        final String sql = getReplacedSql(replacer);
+
+        final List<Object> bindings = new ArrayList<>();
+        final StringBuffer str = new StringBuffer();
+
+        final Pattern pattern = Pattern.compile("\\$\\{(.*?)}");
+        final Matcher matcher = pattern.matcher(sql);
+        if (matcher.find()) {
+            final String name = matcher.group(1);
```
Comment (andriusvelykis) [inline basestar-schema/src/main/java/io/basestar/schema/from/FromSql.java:183]: is this replacement of the _first_ argument only? Should this be `while(matcher.find())`?

**Review by andriusvelykis — APPROVED** [review]
Looks good, just a couple comments

File: basestar-graphql/src/test/resources/io/basestar/graphql/schema.yml (hunk 1 above)
Comment (arcneon) [reply basestar-graphql/src/test/resources/io/basestar/graphql/schema.yml:128]: at this level of test we're only testing the GQL schema adaptor, the whole storage layer is mocked out, so doesn't really matter what the SQL is

**Pull request merged**